R/completion.R

```diff
@@ -0,0 +1,16 @@
+install_github_completer <- function(env) {
+  if (completeme::current_function(env) == "install_github" && completeme::inside_quotes(env)) {
+    res <- jsonlite::fromJSON(paste0("http://rpkg-api.gepuro.net/rpkg?q=", env$token))
```

Comment:
I wonder if we should also support "backwards" completion so that `completeme` could autocomplete to `jimhester/completeme`